node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -192,20 +192,25 @@ impl<I> Era<I> {
     pub(crate) fn resolve_validity(
         &mut self,
         proto_block: &ProtoBlock,
+        timestamp: Timestamp,
```

Comment:
This is a timestamp of the unit that proposed the `ProtoBlock`?

Reply:
Yes

Reply:
…and it will be the timestamp of the `FinalizedBlock` and `Block` itself.

Reply:
Can you add a comment or improve the name? In order to figure out what the `timestamp` is here one has to go up a couple of levels and then down into another branch where it was created.